earl/src/main/java/com/einmalfel/earl/RSSItem.java

```diff
@@ -181,6 +195,10 @@ public String getDescription() {
     return null;
   }
 
+  public String getContentEncoded() {
```

Comment:
I've designed Earl API as follows: if user needs common feed/item attributes and doesn't care about details, (s)he uses getters in Feed/Item interfaces. Otherwise, (s)he accesses fields of Feed/Item implementations. That's why I found this method not fitting the API well. If you insist, let's leave it here, but please add @Nullable then